Formula/brew-php-switcher.rb

```diff
@@ -0,0 +1,27 @@
+class BrewPhpSwitcher < Formula
+  desc "Switch Apache / Valet / CLI configs between PHP versions"
+  homepage "https://github.com/philcook/php-switcher"
+  url "https://github.com/philcook/brew-php-switcher/archive/v2.0.tar.gz"
+  sha256 "c2303b1b1a66ee90ed900c3beabacd6aa4e921dbcad5242e399c45d86899bc88"
+  head "https://github.com/philcook/brew-php-switcher.git"
+
+  def install
+    bin.install "phpswitch.sh"
+    sh = libexec + "brew-php-switcher"
+    sh.write("#!/usr/bin/env sh\n\nsh #{bin}/phpswitch.sh $*")
+    chmod 0755, sh
+    bin.install_symlink sh
+  end
+
+  def caveats; <<~EOS
+    To run brew php switcher do the following:
+      "brew-php-switcher 7.2".
+
+    You can select php version 5.6, 7.0, 7.1, or 7.2.
+    EOS
+  end
+
+  test do
+    system "#{bin}/brew-php-switcher"
```

Comment:
Can you add a test here to see if it can actually switch between versions?

Reply:
What would be the best way to approach that? Considering this relies on whether or not php is installed, if this formula is installed before any other PHP versions, the script will just do nothing. I’m not familiar with testing formulas, so is it possible to install formulas just for testing?

Reply:
yeah, you can do `depends_on "php" => :test` for that.

Reply:
I've added a test, but I'm not sure if it will actually work. I can't get any way to confirm the version actually switched. Is Brew preventing system changes?